apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -49,22 +81,443 @@ export const getPageInfo: GetPageNavInfo = ({
+          defaultMessage: "Edit skill requirements",
+          id: "svskw3",
+        })}
+      >
+        <div data-h2-margin-top="base(-x2)">
+          {categorizedEssentialSkills[SkillCategory.Technical]?.map(
+            (requiredTechnicalSkill) => (
+              <SkillTree
+                key={requiredTechnicalSkill.id}
+                skill={requiredTechnicalSkill}
+                experiences={experiences}
+                showDisclaimer
+                hideConnectButton
+                hideEdit
+              />
+            ),
+          )}
+        </div>
+      </ReviewSection>
+      {screeningQuestions.length > 0 && (
+        <ReviewSection
+          title={intl.formatMessage({
+            defaultMessage: "Screening questions",
+            id: "/7P3+F",
+          })}
+          path={editPaths.screeningQuestions}
+          editLinkAriaLabel={intl.formatMessage({
+            defaultMessage: "Edit screening questions",
+            id: "sNOjXZ",
+          })}
+        >
+          {screeningQuestionResponses.length > 0 ? (
+            <div>
+              <p data-h2-margin-bottom="base(x1)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "You’ve answered the following screening questions:",
+                  id: "GJc/j5",
+                })}
+              </p>
+              <ul>
+                {screeningQuestionResponses.map((response) => (
+                  <li key={response.id} data-h2-margin-bottom="base(x1)">
+                    <p
+                      data-h2-font-weight="base(700)"
+                      data-h2-margin-bottom="base(x.5)"
+                    >
+                      {response.screeningQuestion?.question
+                        ? response.screeningQuestion.question[locale]
+                        : ""}
+                    </p>
+                    <p>{response.answer}</p>
+                  </li>
+                ))}
+              </ul>
+            </div>
+          ) : (
+            <Well>
+              <p data-h2-text-align="base(center)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "It looks like you haven't answered any screening questions yet.",
+                  id: "+ojoEd",
+                })}
+              </p>
+            </Well>
+          )}
+        </ReviewSection>
+      )}
+
+      <section data-h2-margin-bottom="base(x3)">
+        <Heading level="h3" data-h2-margin="base(0, 0, x1, 0)">
+          {intl.formatMessage({
+            defaultMessage: "Sign and submit",
+            id: "+sWqVY",
+          })}
+        </Heading>
+        <div>
+          <FormProvider {...methods}>
+            <form onSubmit={methods.handleSubmit(handleSubmit)}>
+              <p data-h2-margin-bottom="base(x1)">
+                {intl.formatMessage({
+                  defaultMessage: `"You made it! By signing your name, you confirm that:"`,
```

Comment:
Why is this between quotes?

Reply:
😅 Fixed by [04cd097](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/04cd097ac0488a7c7b73635a8d52ea454f945b16)